src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -2888,18 +2884,19 @@ public ConstructorEdit(INamedTypeSymbol oldType)
                                     // If a constructor changes from including initializers to not including initializers
                                     // we don't need to aggregate syntax map from all initializers for the constructor update semantic edit.
                                     var isConstructorWithMemberInitializers = IsConstructorWithMemberInitializers(newDeclaration);
+                                    var isDeclarationWithInitializer = IsDeclarationWithInitializer(oldDeclaration) || IsDeclarationWithInitializer(newDeclaration);
 
-                                    if (isConstructorWithMemberInitializers ||
-                                        IsDeclarationWithInitializer(oldDeclaration) ||
-                                        IsDeclarationWithInitializer(newDeclaration))
+                                    if (isConstructorWithMemberInitializers || isDeclarationWithInitializer)
                                     {
                                         if (isConstructorWithMemberInitializers)
                                         {
                                             processedSymbols.Remove(newSymbol);
                                         }
 
-                                        // Need to check for attribute rude edits for fields and properties
-                                        AnalyzeCustomAttributes(oldSymbol, newSymbol, capabilities, diagnostics, semanticEdits, syntaxMap, cancellationToken);
+                                        if (isDeclarationWithInitializer)
+                                        {
+                                            AnalyzeSymbolUpdate(oldSymbol, newSymbol, newDeclaration, capabilities, diagnostics, semanticEdits, syntaxMap, cancellationToken);
```

Comment:
I'm sure this is covered by some other case (and hopefully tests), but this looks like an issue. Whether there is an initializer, we still need to report rude edits for attributes etc., right?

Reply:
We call DeferConstructor edit for consturctors that emit member initializers and declarations with initializers. This takes care of handling symbol changes (attributes, modifiers) for the later. The former is handled in AddConstructorEdits where we actually issue edits for the constructors.